src/common/wal/src/config/kafka/common.rs

```diff
@@ -25,9 +25,9 @@ use snafu::{OptionExt, ResultExt};
 /// The default backoff config for kafka client.
 pub const DEFAULT_BACKOFF_CONFIG: BackoffConfig = BackoffConfig {
     init_backoff: Duration::from_millis(100),
-    max_backoff: Duration::from_secs(10),
-    base: 2.0,
-    deadline: Some(Duration::from_secs(120)),
+    max_backoff: Duration::from_secs(1),
+    base: 3.0,
+    deadline: Some(Duration::from_secs(3)),
```

Comment:
Please add comments for these configurations.

Reply:
resolved in [4f58a40](https://github.com/GreptimeTeam/greptimedb/pull/6092/commits/4f58a40c78c4e8194176e271b02e422f67e26d8d)

Reply:
Are max_backoff and deadline meaning the same?

Reply:
The `max_backoff` represents the max wait time between retry attempts, while deadline represents the total time limit for all retry operations.